cmd/check.go

```diff
@@ -0,0 +1,117 @@
+// Copyright © 2017 Mesosphere Inc. <http://mesosphere.com>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"math"
+	"os"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/dcos/3dt/runner"
+	"github.com/spf13/cobra"
+	"strings"
+)
+
+const (
+	checkTypeCluster       = "cluster"
+	checkTypeNodePreStart  = "node-prestart"
+	checkTypeNodePostStart = "node-poststart"
+)
+
+var (
+	list          bool
+	checksCfgFile string
+)
+
+// checkCmd represents the check command
+var checkCmd = &cobra.Command{
+	Use:   "check <check-type>",
+	Short: "Execute a DC/OS check",
+	Long: `A DC/OS check can be one of the following types: cluster, node-prestart, node-poststart
+
+cluster ...
+node-prestart ...
+node-poststart ...
+
+	`,
+	Run: func(cmd *cobra.Command, args []string) {
+		var checkNames []string
+		if len(args) == 0 {
+			cmd.Usage()
+			return
+		} else if len(args) > 1 {
+			checkNames = args[1:]
+		}
+
+		r := runner.NewRunner(defaultConfig.FlagRole)
+		if err := r.LoadFromFile(checksCfgFile); err != nil {
+			logrus.Fatal(err)
+		}
+
+		var (
+			rs  []runner.Response
+			err error
+		)
+
+		switch args[0] {
+		case checkTypeCluster:
+			rs, err = r.Cluster(nil, list, checkNames...)
```

Comment:
Does it make sense to use `context.Background()` here and below instead of passing in nil?